core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -84,14 +85,15 @@ object DagrCoreMain extends Configuration {
     // the method that should be called once the dagr and pipeline instances have been successfully created
     def clpBlock(dagr: DagrCoreMain)(pipeline: Pipeline): Unit = dagr.configure(pipeline)
 
-    parser.parseCommandAndSubCommand[DagrCoreMain](
-      args                = args,
-      packageList         = packageList,
-      omitSubClassesOf    = Seq(classOf[DagrCoreMain]),
-      includeHidden       = includeHidden,
-      afterCommandBuild           = mainBlock,
-      afterSubCommandBuild            = clpBlock
+    val mainAndPipeline = parser.parseCommandAndSubCommand[DagrCoreMain](
+      args                 = args,
+      packageList          = packageList,
+      omitSubClassesOf     = Seq(classOf[DagrCoreMain]),
+      includeHidden        = includeHidden,
+      afterCommandBuild    = mainBlock,
+      afterSubCommandBuild = clpBlock
     )
+    yieldAndThen(mainAndPipeline) { mainAndPipeline.map(_._2).filter(_.printPipelineArgs).foreach { p => System.err.println(args.mkString(" ")) } }
```

Comment:
If I'm understanding this correctly, this will only print the args from the command line, and not any defaults that came from elsewhere, right?  So I think that because of where this is anything in and `@argfile` will not be present (just the literal `@argfile`) and any defaults from the class itself.

I was hoping you'd find some convenient place in `sopt` to construct a string that contained all the parameter value used to instantiate the class.